test/runtests.jl

```diff
@@ -23,22 +16,28 @@ using Random
 using StaticArrays
 using Statistics
 using Test
+using JuliaInterpreter
 
-@nospecialize
-
-if isdefined(Base, :Experimental) && isdefined(Base.Experimental, Symbol("@compiler_options"))
-    @eval Base.Experimental.@compiler_options compile=min optimize=0 infer=no
-end
+union!(JuliaInterpreter.compiled_modules, Any[Base, Base.Broadcast, Compat, LinearAlgebra, Random, StaticArrays, Statistics])
 
 Random.seed!(1) # Set seed that all testsets should reset to.
 
-function include_test(path::String)
+function include_test(path::String; interpret=true)
```

Comment:
we seem to never set `interpret=false`.
We would just remove this option.

Reply:
What if JulieInterpreter will have future problems on, e.g., nightly, making testing ChainRules impossible until JI is working again?

Reply:
I previously had this because I observed some tests being faster this way, but those went away by letting some more modules be compiled, so I might as well delete it.

> What if JulieInterpreter will have future problems on, e.g., nightly, making testing ChainRules impossible until JI is working again?

Guess someone will have to fix JuliaInterpreter then. :) Usually we fix those pretty fast though.

Reply:
> What if JulieInterpreter will have future problems on, e.g., nightly, making testing ChainRules impossible until JI is working again?

Editting `include_test(x)` to  say `@time include(x)` would  take for all intents and purposes as long as editting `interpret=true` to `interpret=false`.